### DISCUSSION TITLE
Album release dates

### DISCUSSION THREAD

**Discussion by Sevinfolds** [navidrome/navidrome#3341] [Ideas]
How do you feel about adding the year of release to the album card? The year of release is only displayed if you go to the artist, on the main page, or in the search there is no such thing. It would be convenient if it was on other pages as well, because it is not always clear whether it is an old album or a new one. (I indicated it in red as an example)
![Screenshot_7](https://github.com/user-attachments/assets/696f2799-9766-4110-a728-a943036e6401)

**Comment by Kristoffeh** [comment]
I agree that there should be a simpler way to see when an album was released.
But i do not think that including it in the title of the album is a good idea.

But could do something like:

```
Long Time, No Sea
Attack Attack!
2024
```

In case it is not clear enough for others:
```
<album>
<artist>
<release_date>
```

But of course: I think this `feature` should be at the bottom of the list of priorities, it really isn't that big of a deal.
